cinema/views.py

```diff
@@ -55,4 +86,57 @@ def get_serializer_class(self):
         if self.action == "retrieve":
             return MovieSessionDetailSerializer
 
-        return MovieSessionSerializer
+        return self.serializer_class
+
+    @staticmethod
+    def _params_to_ints(qs) -> list:
+        return [int(str_id) for str_id in qs.split(",")]
```

Comment:
u have already implemented this method above, so make ur code DRY - u can write this method as standalone func inside ur views